app/jobs/release_episodes_job.rb

```diff
@@ -1,15 +0,0 @@
-class ReleaseEpisodesJob < ApplicationJob
```

Comment:
As far as I can tell - this hasn't been used for 7+ years?  This was the old style SQS jobs we used to use, that would "reschedule" themselves.

But these days, our episode releasing is via `config/initializers/say_when.rb` - which calls `Podcast.release!` every 5 minutes.